pkg/util/mountOpts.go

```diff
@@ -33,6 +33,10 @@ func ProcessOptions(options []string, isTmpfs bool, sourcePath string) ([]string
 		// Some options have parameters - size, mode
 		splitOpt := strings.SplitN(opt, "=", 2)
 		switch splitOpt[0] {
+		case "idmap":
+			if len(options) > 1 {
+				return nil, errors.Wrapf(ErrDupeMntOption, "'idmap' option can not be used with other options")
```

Comment:
Why is this?

Reply:
I've copied the same logic as `O`.

I'll drop it

Reply:
I am just wondering if this would be an issue with Z, or RO?

Reply:
`-v foo:/foo:idmap,Z` seems to work fine